README.md

```diff
@@ -72,16 +65,28 @@ by this point.
     cd NAB
     (sudo) pip install -r requirements.txt
 
-This will install the requirements, such as pandas and simplejson.  
+This will install the additional required modules pandas and simplejson.  
 
 
 ### Usage
 
+##### Run NAB
+
     cd /path/to/nab
-    python setup.py develop (for now)
```

Comment:
Yes you're right. Do you know who put the `(for now)` part, and what we intend to use instead?